pippo-core/src/main/java/ro/pippo/core/route/ClasspathResourceHandler.java

```diff
@@ -32,10 +38,20 @@ public ClasspathResourceHandler(String urlPath, String resourceBasePath) {
         this.resourceBasePath = getNormalizedPath(resourceBasePath);
     }
 
+    private static Path ROOT_PATH = Paths.get("/");
+    private static String normalizePath(String subPath) {
+        return ROOT_PATH.relativize(ROOT_PATH.resolve(subPath).normalize()).toString();
+    }
+
     @Override
     public URL getResourceUrl(String resourcePath) {
-        String resourceName = getResourceBasePath() + "/" + resourcePath;
-        return this.getClass().getClassLoader().getResource(resourceName);
+        String normalizedPath = normalizePath(resourcePath);
```

Comment:
Is it not much simple to have a method:
```
// we can choose other name, if you feel that actual name is not suggestive
boolean validateResourcePath(String root, String path) {
    // put the logic here
}
```

and use this method in both `ClasspathResourceHandler` and `WebjarsResourceHandler`?